docs/ipfs.md

```diff
@@ -124,10 +132,17 @@ FROM localhost:5050/ipfs/bafkreicq4dg6nkef5ju422ptedcwfz6kcvpvvhuqeykfrwq5krazf3
 RUN echo hello > /hello
 ```
 
-You can enable builds on IPFS using `--ipfs` option for `nerdctl build`.
+Make sure that `nerdctl ipfs registry serve` is running.
+This allows `nerdctl build` to pull images from IPFS.
+
+```
+$ nerdctl ipfs registry serve &
+```
+
+Then you can build this Dockerfile using `nerdctl build`.
 
 ```console
-> nerdctl build --ipfs -t hello .
```

Comment:
The document may note that this flag (and up/down) is being removed in nerdctl v1.2.0.